pkgs/top-level/all-packages.nix

```diff
@@ -17107,8 +17107,6 @@ in
 
   randomx = callPackage ../development/libraries/randomx { };
 
-  redkite = callPackage ../development/libraries/redkite { };
```

Comment:
Please add an alias to aliases.nix.

Reply:
You mean a message saying it's been removed?
Redkite was written for geonkick so I'm pretty sure no one has or wants it installed standalone.

Reply:
> You mean a message saying it's been removed?

yes.

> Redkite was written for geonkick so I'm pretty sure no one has or wants it installed standalone.

Please do it anyway. :)

Reply:
Here you go! :)